Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -206,6 +208,10 @@ public void onAdEnd(@NonNull String placementReferenceId, boolean wasSuccessfulV
                 mIsPlaying = false;
                 sVungleRouter.removeRouterListener(mPlacementId);
                 mVungleRouterListener = null;
```

Comment:
not related to the changes here but seems like we dont uniformly follow on what thread we are making these callbacks, some have `mHandler.post` and some do not, could this create issues with Pub getting different events posted back on different threads

Reply:
ok, fixed it follow main UI thread for MoPub callbacks